lib/ansible/modules/network/avi/avi_analyticsprofile.py

```diff
@@ -30,130 +30,154 @@
+        type: float
     apdex_server_rtt_threshold:
         description:
             - Satisfactory client to avi round trip time(rtt).
             - Allowed values are 1-2000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 125.
+        type: int
     apdex_server_rtt_tolerated_factor:
         description:
             - Tolerated client to avi round trip time(rtt) factor.
             - It is a multiple of apdex_rtt_tolerated_factor.
             - Allowed values are 1-1000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 4.0.
+        type: float
     client_log_config:
         description:
             - Configure which logs are sent to the avi controller from ses and how they are processed.
+        type: dict
     client_log_streaming_config:
         description:
             - Configure to stream logs to an external server.
             - Field introduced in 17.1.1.
         version_added: "2.4"
+        type: dict
     conn_lossy_ooo_threshold:
         description:
-            - A connection between client and avi is considered lossy when more than this percentage of out of order packets are received.
+            - A connection between client and Avi is considered lossy when more than this percentage of out of order packets are received.
             - Allowed values are 1-100.
             - Default value when not specified in API or module is interpreted by Avi Controller as 50.
+        type: int
     conn_lossy_timeo_rexmt_threshold:
         description:
-            - A connection between client and avi is considered lossy when more than this percentage of packets are retransmitted due to timeout.
+            - A connection between client and Avi is considered lossy when more than this percentage of packets are retransmitted due to timeout.
             - Allowed values are 1-100.
             - Default value when not specified in API or module is interpreted by Avi Controller as 20.
+        type: int
     conn_lossy_total_rexmt_threshold:
         description:
-            - A connection between client and avi is considered lossy when more than this percentage of packets are retransmitted.
+            - A connection between client and Avi is considered lossy when more than this percentage of packets are retransmitted.
             - Allowed values are 1-100.
             - Default value when not specified in API or module is interpreted by Avi Controller as 50.
+        type: int
     conn_lossy_zero_win_size_event_threshold:
         description:
-            - A client connection is considered lossy when percentage of times a packet could not be transmitted due to tcp zero window is above this threshold.
+            - A client connection is considered lossy when percentage of times a packet could not be trasmitted due to TCP zero window is above this threshold.
```

Comment:
Original spelling of `transmitted` was correct:
```suggestion
            - Threshold for defining a lossy connection by packets not transmitted due to window size.
            - A client connection is considered lossy when percentage of times a packet could not be transmitted due to TCP zero window is above this threshold.
```